<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>630</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <underline>false</underline>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="Carcteristicas">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>290</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Caracteristicas</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="txt_UmbralSVM">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Umbral SVM</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_UmbralRNN">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>95</y>
       <width>120</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Umbral RNN</string>
     </property>
    </widget>
    <widget class="QLabel" name="Time_Sequence">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Time_Pose">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>95</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSpinBox" name="UmbralSVM">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>42</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="UmbralRNN">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>90</y>
       <width>42</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Det_Capture">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>460</y>
      <width>310</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Control Grabacion</string>
    </property>
    <widget class="QPushButton" name="Start_Pose">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>50</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Empezar Secuencia</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Stop_Pose">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>90</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Ver Pose</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="Count_Box">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>80</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="countdownTime">
      <property name="geometry">
       <rect>
        <x>29</x>
        <y>38</y>
        <width>22</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="Img_Record">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>20</y>
        <width>60</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="text">
       <string>⬤</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Img_Play">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>22</y>
        <width>60</width>
        <height>65</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>50</pointsize>
       </font>
      </property>
      <property name="text">
       <string>▶</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Img_Pause">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>25</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>50</pointsize>
       </font>
      </property>
      <property name="text">
       <string>■</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Prediction_Box">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>460</y>
      <width>290</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Prediccion Movimiento</string>
    </property>
    <widget class="QLabel" name="Move_text">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Movimiento</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="Move_BackGround">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>45</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Type_Prediction_Text">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>95</y>
       <width>95</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Prediccion</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="Type_Prediction">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>88</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Move">
     <property name="geometry">
      <rect>
       <x>155</x>
       <y>50</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>____________</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Camera_Img">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>440</width>
      <height>410</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="Sequence">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>400</width>
       <height>350</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <underline>false</underline>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Pose_Img">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>440</width>
      <height>410</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="View_Pose">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>400</width>
       <height>350</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <underline>false</underline>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="load">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>371</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="load_text">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>160</y>
      <width>181</width>
      <height>65</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>87</green>
          <blue>83</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>136</red>
          <green>138</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>204</red>
          <green>207</green>
          <blue>200</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Midlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>170</red>
          <green>172</green>
          <blue>166</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Dark">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>68</red>
          <green>69</green>
          <blue>66</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Mid">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>91</red>
          <green>92</green>
          <blue>89</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>87</green>
          <blue>83</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="BrightText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>87</green>
          <blue>83</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>136</red>
          <green>138</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Shadow">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="AlternateBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>195</red>
          <green>196</green>
          <blue>194</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ToolTipBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>220</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ToolTipText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>87</green>
          <blue>83</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>136</red>
          <green>138</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>204</red>
          <green>207</green>
          <blue>200</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Midlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>170</red>
          <green>172</green>
          <blue>166</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Dark">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>68</red>
          <green>69</green>
          <blue>66</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Mid">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>91</red>
          <green>92</green>
          <blue>89</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>87</green>
          <blue>83</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="BrightText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>87</green>
          <blue>83</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>136</red>
          <green>138</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Shadow">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="AlternateBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>195</red>
          <green>196</green>
          <blue>194</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ToolTipBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>220</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ToolTipText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>87</green>
          <blue>83</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>136</red>
          <green>138</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>204</red>
          <green>207</green>
          <blue>200</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Midlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>170</red>
          <green>172</green>
          <blue>166</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Dark">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>68</red>
          <green>69</green>
          <blue>66</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Mid">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>91</red>
          <green>92</green>
          <blue>89</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>87</green>
          <blue>83</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="BrightText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>87</green>
          <blue>83</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>136</red>
          <green>138</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>136</red>
          <green>138</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Shadow">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="AlternateBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>136</red>
          <green>138</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ToolTipBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>220</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ToolTipText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Samyak Tamil</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 87, 83);</string>
    </property>
    <property name="text">
     <string>Conectando Simulador</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QColumnView" name="Background_Load">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>362</x>
      <y>120</y>
      <width>271</width>
      <height>231</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);
border: 2px solid rgb(222, 222, 222);;
border-radius: 10px;</string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
   </widget>
   <zorder>Pose_Img</zorder>
   <zorder>Camera_Img</zorder>
   <zorder>Background_Load</zorder>
   <zorder>Carcteristicas</zorder>
   <zorder>Det_Capture</zorder>
   <zorder>Prediction_Box</zorder>
   <zorder>load</zorder>
   <zorder>load_text</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
